MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.png)
+
+![Print Preview Window](Images\Print\print-preview-window.png)
+
+
+Using [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html), you can effortlessly print PDF documents directly from your application code. This feature enables users to conveniently fill PDF form fields and print the document with the updated values incorporated.
+
+Moreover, when performing a print operation, the default system print dialog opens, providing users with familiar options and settings to customize their printing experience. Within this dialog, users can adjust settings such as printer selection, the number of copies, page orientation, and more, ensuring that their printed documents meet their specific requirements.
+
+To print a PDF programmatically, you can use the `PrintDocument()` method provided by SfPdfViewer. This method enables you to trigger the printing action seamlessly within your application.
```

Comment:
No need to mention brackets as you mentioned as method on its followup.
<img width="190" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/0514f778-773a-4cba-b175-7179a37c1b76">

Reply:
Removed the brackets in method.